apps/avifenc.c

```diff
@@ -994,7 +994,7 @@ int main(int argc, char * argv[])
 
     if (gridDimsCount > 0) {
         avifResult addImageResult = avifEncoderAddImageGrid(
-            encoder, (uint8_t)gridDims[0], (uint8_t)gridDims[1], (const avifImage **)gridCells, AVIF_ADD_IMAGE_FLAG_SINGLE);
+            encoder, (uint8_t)gridDims[0], (uint8_t)gridDims[1], (void*)gridCells, AVIF_ADD_IMAGE_FLAG_SINGLE);
```

Comment:
Hi Pascal: Thank you for writing a fix! Casting to the intermediate void* is not ideal. I like Yuan Tong's suggestion of adding one more const to the declaration of the gridCells parameter of avifEncoderAddImageGrid(). I verified it works.

Reply:
Wan-Teh, Yuan:  yes, the c-faq description of the problem was the one i was referring to as "obscure C rule".
I thought using "void*" was an easy way for users to not ask questions they probably don't really want to know the answer of ;)

I'm totally fine with doing it the explicit way! ("const ptr* const * ...")